sycl/test-e2e/KernelCompiler/kernel_compiler_sycl.cpp

```diff
@@ -45,7 +52,7 @@ void ff_templated(T *ptr) {
   sycl::nd_item<1> Item = sycl::ext::oneapi::this_work_item::get_nd_item<1>();
 
   sycl::id<1> GId = Item.get_global_id();
-  ptr[GId.get(0)] = GId.get(0) + 39;
+  ptr[GId.get(0)] = PlusEm(GId.get(0), 38);
```

Comment:
Why `39` replaced with `38`?

Reply:
because I'm introducing PlusEm, which uses a different value than AddEm and didn't want to go and change all the asserts.

Reply:
Thanks @dm-vodopyanov !  There was a mistake in the test assertions and your question made me realize it existed. Fixed.